### ISSUE TITLE
Weird cache problem

### ISSUE THREAD

**Issue by ponychicken** [kolber/stacey#22]
On two Stacey site I have the followin problem: After a while, and with no changes on my site the cache for one or more pages is corrupted. All paths which should be prefaced by "../" are now only prefaced by "./". All refereced css/js/links are then 404's

**Comment by papillon-unix** [comment]
yes, I have exactly the same problem, it's really disturbing. The only way to fix it is to open the .yml file of the page and to save it again without any change. But... it does happens again after a while, and you never can predict on which page it will occurs..
Help !!

**Comment by joaoribeirovaz** [comment]
I also have this problem. I found that not using the SLIR (image resizing parser) to output large images reduces _most_ cache problems.

(when the problem occurs, I usually save the htacess file)
(I work on a I site with > 150 pages, so to find this error I use brokenlinkcheck.com)

**Comment by Mark2Mark** [comment]
I have the same thing happening. I usually rename the containing folder forth and back to fix it.

I wonder if there’s a way to inform yourself that a site is corrupted?
Currently I can only rely on users reporting me the broken sites.
